examples/combined_ODNP.py

```diff
@@ -150,6 +167,7 @@
         filename = "temp_ctrl.h5"
         DNP_data.hdf5_write(filename, directory=target_directory)
         final_log.append("change the name accordingly once this is done running!")
+    raise    
```

Comment:
This causes it to quit

Reply:
so we don't want `raise` since if there is an error, this will be thrown forcing the script to stop running. Is what I did with `except Exception:` okay?